### ISSUE TITLE
sis671 loop

### ISSUE THREAD

**Issue by agarciamontoro** [gununu/xf86-video-sis-stable#2]
Hi,

I'm having problems with a SIS671 card on an ArchLinux system. I have installed your driver along with gnome+gdm (I've also tested lxdm) but the system enters in an endless loop when starting the graphic manager.

The screen flicks between a black screen and a black screen with a cursor at the topleft corner.

Have you ever encountered this issue? Thank you.

**Comment by fu-sen** [comment]
I recently had the same experience with Slackware.
(Before that I encounter errors with make)
Probably the current sources to Ubuntu and Debian only,
others distribution will need to provide patches.

**Comment by agarciamontoro** [comment]
I also encountered errors with make before using this driver: videomodeproc.h reference was deleted here and that solved this compilationerror.

I am trying to use other xorg.conf, as the one referenced here: http://askubuntu.com/a/504632 This solve the loop problem, but there are *a lot* of visual artifacts that make the graphic sysetm unusable.

**Comment by fu-sen** [comment]
Hmm? Do you modify and use xorg.conf? The driver is not applied in this.

**Comment by fu-sen** [comment]
Slackware got an error separately from videomodeproc.h.
Perhaps it may be another problem.

**Comment by agarciamontoro** [comment]
Sorry, that was the vesa one, which I also tested. I also tried the changes on https://wiki.archlinux.org/index.php/SiS#SiS_671_card, but nothing worked.
What do you mean by "driver is not applied in this"?

**Comment by fu-sen** [comment]
If this is also the case, this driver will not apply!
You refer to the README once and understand it:
https://github.com/gununu/xf86-video-sis-stable/blob/master/README

**Comment by agarciamontoro** [comment]
Yep, that was the first thing tested, when the flickering loop occured. I then tried adding the UserTiming option and finally changing all the file.

Anyway, the issue is still there: with the configuration from the README, there is a flickering loop when starting the graphic manager.

**Comment by MikalaiR** [comment]
Try to disable graphic console (disable gfxpayload in grub).

**Comment by mirh** [comment]
Could this be related?
https://bugs.freedesktop.org/show_bug.cgi?id=98158

**Comment by gununu** [comment]
Hi everyone.
I'd like to inform you that:
I don't have motivation and time to fix this driver except compilation errors...
Sorry.

**Comment by ace2nutzer** [comment]
Hi,
if someone is still interested, i've fixed this issue.
Note: read carefully my README for proper installation method.

https://github.com/ace2nutzer/xf86-video-sis-stable
